content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Kube-scheduler Profiles
+content_template: templates/concept
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.18" state="alpha" >}}
+
+A scheduling Profile allows to configure the different stages of scheduling
+in [kube-scheduler](/docs/reference/command-line-tools-reference/kube-scheduler/).
+Each stage is exposed in a extension point. Plugins provide scheduling behaviors
+by implementing one or more of these extension points.
+
+You can specify scheduling profiles for kube-scheduler in a configuration file
+(via `--config` flag) using the component config APIs
+([`v1alpha1`](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1alpha1?tab=doc#KubeSchedulerConfiguration)
+or [`v1alpha2`](https://pkg.go.dev/k8s.io/kube-scheduler@v0.18.0/config/v1alpha2?tab=doc#KubeSchedulerConfiguration)).
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Extension points
+
+Scheduling stages are exposed in the following extension points:
+
+1. **QueueSort**: These plugins provide an ordering function that is used to
+   sort pending Pods in the scheduling queue. Exactly one queue sort plugin
+   may be enabled at a time.
+1. **PreFilter**: These plugins are used to pre-process or check information
+   about a Pod or the cluster before filtering.
+1. **Filter**: These plugins are the equivalent of Predicates in a scheduling
+   Policy and are used to filter out nodes that can not run the Pod. Filters
+   are called in the configured order.
+1. **PreScore**: This is an informational extension point that can be used
+   for doing pre-scoring work.
+1. **Score**: These plugins provide a score to each node that has passed the
+   filtering phase. The scheduler will then select the node with the highest
+   weighted scores sum.
+1. **Reserve**: This is an informational extension point that notifies plugins
+   when resources have being reserved for a given Pod.
+1. **Permit**: These plugins can prevent or delay the binding of a Pod.
+1. **PreBind**: These plugins perform any work required before a Pod is bound.
+1. **Bind**: The plugins bind a Pod to a Node. Bind plugins are called in order
+   and once one has done the binding, the remaining plugins are skipped. At
+   least one bind plugin is required.
+1. **PostBind**: This is an informational extension point that is called after
+   a Pod has been bound.
+1. **UnReserve**: This is an informational extension point that is called if
+   a Pod is rejected after being reserved and put on hold by a Permit plugin.
+   
+## Scheduling plugins
+
+The following plugins, enabled by default, implement one or more of these
+extension points:
+
+
+- `DefaultTopologySpread`: Favors spreading across nodes for Pods that belong to
```

Comment:
For example, this could change to:

```markdown
 `DefaultTopologySpread`: Favors spreading across nodes for Pods that belong to
  services, replica sets and stateful sets.
  Extension points: *`PreScore`*, *`Score`*.
```

Reply:
I'd expect this list to be in alphabetical order. Is there a different order that is relevant?

Reply:
There is no reasonable order, but alphabetical will prevent us from having the volume plugins together